pages/blog/release-notes-2.3.0.md

```diff
@@ -0,0 +1,116 @@
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.2.0/cover.webp
+authors:
+  - name: Dale Lane
+    photo: /img/avatars/questionmark.webp
+    link: https://twitter.com/dalelane
+excerpt: 'AsyncAPI 2.3 is now released. This brings enhancements allowing new ways of structuring AsyncAPI documents and support for describing Solace APIs.'
+featured: true
+---
+
+The new version of the AsyncAPI specification - 2.3.0 - is now available.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.2.0'` into `asyncapi: '2.3.0'`
+
+
+## Servers and channels can now be defined as reusable components
+
+To allow for more flexibility in how AsyncAPI documents are structured and enable content to be reused, `servers` and `channels` can now be defined as reusable components.
+
+For example:
+
+```yaml
+asyncapi: 2.3.0
+servers:
+  production:
+    $ref: '#/components/servers/myserver'
+channels:
+  some/events:
+    $ref: '#/components/channels/myChannel'
+components:
+  servers:
+    myserver:
+      url: "http://localhost:5000/ws"
+      protocol: ws
+  channels:
+    myChannel:
+      description: "mychannel"
+```
+
+These are added to the many other aspects of the AsyncAPI specification which can be declared as reusable components. You can see the full list in the [Components Object section of the AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.3.0#componentsObject).
+
+This new feature was contributed by [Sergio Moya](https://www.linkedin.com/in/smoya). For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/665) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/660).
+
+
+## Channels are now optional
+
+Related to the above change, and also in support of enabling greater flexibility in how AsyncAPI documents are structured and reused, `channels` are no longer a required section.
+
+This change makes it easier, for example, to have an AsyncAPI document that only contains reusable component definitions.
+
+For example, you could now have a document like this, which you use solely to hold reusable components for use in other documents.
+
+```yaml
+asyncapi: 2.3.0
+info:
+  description: Dictionary for our definitions
+servers:
+  production:
+    $ref: '#/components/servers/myserver'
+components:
+  servers:
+    myserver:
+      url: "http://localhost:5000/ws"
+      protocol: ws
+  channels:
+    myChannel:
+      description: "mychannel"
+    myOtherChannel:
+      description: "another channel"
+```
+
+This could be then used in other documents, like:
+```yaml
+asyncapi: 2.3.0
+info:
+  description: My awesome service API
+servers:
+  myserver:
+    $ref: 'dictionary.yaml#/components/servers/myserver'
+channels:
+  mychannel1:
+    $ref: 'dictionary.yaml#/components/channels/myChannel'
+  mychannel1:
```

Comment:
Hey @dalelane ,

Sorry maybe it's my misunderstanding here, but is `mychannel1` meant to be twice here for your channels example?

Reply:
nope - nice catch, thanks :)

Reply:
```suggestion
  mychannel2:
```